### DISCUSSION TITLE
基于 TensorFlow Lite 的运动识别 | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1013] [Q&A]
# 基于 TensorFlow Lite 的运动识别 | Seeed Studio Wiki

Getting Started

https://wiki.seeedstudio.com/cn/XIAO-BLE-Sense-TFLite-Getting-Started/

**Comment by tianning2001** [comment]
MU_Classifier.ino编译不过，Arduino_TensorFlowLite\src/peripherals/peripherals.h:61:2: error: #error "unsupported board"
 #error "unsupported board"
